project.godot

```diff
@@ -21,6 +21,7 @@ PlayerManager="*res://Managers/PlayerManager.gd"
 CardManager="*res://Managers/CardManager.gd"
 PhaseManager="*res://Managers/PhaseManager.gd"
 RandomGenerator="*res://Managers/RandomGenerator.gd"
+GlobalVar="*res://Global/GLOBAL_VAR.gd"
```

Comment:
I don't think you need to add this to the autoload, we don't do it for the Enums.gd. You will have to add `class_name GLOBAL_VAR` to the `GLOBAL_VAR.gd` though

Reply:
Yeah I was thinking about the `Enums.gd` the other day and was wondering if we should move keep that separate or not ?
I was thinking of putting the files from the enum into the global. That was it's clear that this for global variables.

Reply:
I kind of prefer the Enums file because you know it's only enums and the file name is shorter. But either is fine. I still think you can remove GLOBAL_VAR from the Autoload though. I'll go ahead and approve though since I know you're trying to finish this up